src/lib/api.ts

```diff
@@ -9,11 +9,28 @@ if (!apiBaseUrl) {
 export const fetchData = async (path: string) => {
   try {
     const response = await axios.get(`${apiBaseUrl}/${path}`);
+    const footerData = await fetchFooter();
+
     if (response.status !== 200) {
       throw new Error('Failed to fetch data');
     }
-    return response.data;
+    return { data: response.data, footer: footerData };
   } catch (error) {
     throw new Error('Failed to fetch data');
   }
 };
+
+export const fetchFooter = async () => {
+  try {
+    const response = await axios.get(
+      'https://wcc-backend.fly.dev/api/cms/v1/footer',
```

Comment:
good idea here is also to create a variable and reuse, because this can change, the host and possible the api path: `api/cms/v1/footer`

Reply:
this will be changed in the next PR (I'm replacing the mock landing with the actual backend so API_BASEURL will be holding the URL for both footer and landing (and others))